test/02_GiriGiriBashi.spec.ts

```diff
@@ -279,7 +279,7 @@ describe("GiriGiriBashi", function () {
   })
 
   describe("getHash()", function () {
-    it("Reverts if threshold is not met", async function () {
+    it("Reverts if threshold is not set", async function () {
```

Comment:
In this case, "not met" is correct. This is testing to make sure that the transaction reverts if the number of adapters is lower than the threshold for the given domain.

Reply:
Agreed!